### PR TITLE
07: Nav Bar

### PR DESCRIPTION
**Pull request by gabrielizalo**
# Summay

* Updated html and scss files adding Nav Bar.

# Trello Card

https://trello.com/c/nqenCdTY/13-navbar

# Screenshot

screenshot 2018-10-02_14-27-59-089

### CODE REVIEW

File: maquetado/scss/_navbar.scss [hunk 1]
```diff
@@ -0,0 +1,23 @@
+.nav {
+  align-items: center;
+  background-color: $white;
+  box-shadow: 0 0 15px #cfcfcf;
```
Comment (mcallegari10) [inline maquetado/scss/_navbar.scss]: Use variables for colors 😄

File: maquetado/index.html [hunk 2]
```diff
@@ -9,14 +9,71 @@
 </head>
 <body>
   <!-- NAV -->
-  <!-- TODO -->
+  <nav class="nav">
+    <img alt="Wolox" class="nav-logo" src="assets/wolox_logo.svg">
```
Comment (mcallegari10) [inline maquetado/index.html:13]: Use an `alt` attribute please

File: maquetado/scss/_navbar.scss [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  display: flex;
+  justify-content: space-between;
+  left: 0;
+  position: fixed;
+  top: 0;
+  width: 100%;
+  z-index: 10;
```
Comment (henryzarza) [inline maquetado/scss/_navbar.scss]: I think that with a value of `1` or `2` is enough. It's not the best practice to set high `z-index` values

**Review by henryzarza — CHANGES_REQUESTED** [review]

File: maquetado/index.html (hunk 2 above)
Comment (gabrielizalo) [reply maquetado/index.html:13]: The alt is present... Does it has to be at the end?

File: maquetado/scss/_tech.scss [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+    left: -10px;
+    position: absolute;
+    top: -10px;
+    transition: transform 500ms ease;
+    width: 100%;
+    z-index: -1;
+  }
+
+  &:hover:before {
+    transform: translate(20px, 20px);;
```
Comment (henryzarza) [inline maquetado/scss/_tech.scss]: You have two `;` delete one

**Review by henryzarza — DISMISSED** [review]
Great! You look my comment

**Review by henryzarza — APPROVED** [review]

**Pull request merged**